ground/gcs/src/plugins/coreplugin/telemetrymonitorwidget.cpp

```diff
@@ -216,7 +217,7 @@ void TelemetryMonitorWidget::resizeEvent(QResizeEvent* event)
 
     // This offset is required because the widget is not centered while
     // it is a child of the connection selector widget
-    graph->setPos(0,-20);
+    //graph->setPos(0,-20);
```

Comment:
In future you can just delete code rather than commenting it out unless it adds some explanation. :wink: